### DISCUSSION TITLE
5 pin megadesk controller.  Which pins are is SCK and MISO

### DISCUSSION THREAD

**Discussion by nakermann1973** [gcormier/megadesk#153] [General]
I am building my own esp8266 companion, and am reasy to start connecting things up.  So far I have not found a description of the 5 pin header of the newer megadesk boards though.  Is the documentation out of date?

I am assuming that I connect SCK to TX and MISO to RX.  Is this correct?

**Comment by gcormier** [comment]
Pin 5 is RX (PA5/MISO), where Megadesk expects to receive serial data. Pin 4 is TX (PA4/SCK).
